internal/indexer/postgres/store.go

```diff
@@ -92,6 +97,11 @@ func (s *store) IndexReport(ctx context.Context, hash claircore.Digest) (*clairc
 	return sr, b, err
 }
 
+func (s *store) AffectedManifests(ctx context.Context, v claircore.Vulnerability) ([]claircore.Digest, error) {
+	hashes, err := affectedManifests(ctx, s.pool, v)
+	return hashes, err
```

Comment:
```suggestion
	return affectedManifests(ctx, s.pool, v)
```

Reply:
this is following the pattern of all other methods in that file.